components/expenses/ExpenseForm.js

```diff
@@ -833,7 +834,13 @@ const ExpenseForm = ({
   const isDraft = expense?.status === expenseStatus.DRAFT;
   const [hasValidate, setValidate] = React.useState(validateOnChange && !isDraft);
   const intl = useIntl();
-  const initialValues = { ...getDefaultExpense(collective), ...expense };
+  const supportedExpenseTypes = React.useMemo(() => getSupportedExpenseTypes(collective), [collective]);
+  const isSingleSupportedExpenseType = supportedExpenseTypes.length === 1;
+  const initialValues = {
+    ...getDefaultExpense(collective),
+    type: !isDraft && isSingleSupportedExpenseType ? supportedExpenseTypes[0] : undefined,
```

Comment:
Why are we ignoring drafts, are they causing any issues?

Reply:
No, I removed `!isDraft` from condition